ui/raidboss/data/07-dt/raid/r2n.txt

```diff
@@ -1,8 +1,77 @@
 ### Arcadion (R2N): AAC Light-heavyweight M2
+# -ii 917E 91C7 9A65 9166 9168 9172 9A6A 9165 916B 916C 916E 917C 916D 917F 9A66 9169
+# -it "Honey B. Lovely"
+
+# 9163 = teleport to middle
 
 hideall "--Reset--"
 hideall "--sync--"
 
-0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
```

Comment:
Haven't had a chance to look at this (and probably won't be able to tonight) - but did have a question about this, mostly for my own edification.  Are explicit reset entries no longer required?  I think they're in all of the pandaemonium raid files, but perhaps there was a change at some point? Or the timeline controller resets when out of combat?

Reply:
I'm not 100% sure off the top of my head. I copy/pasted the timeline from the `make_timeline` tool initially, apparently that doesn't include the reset. If it's still required, I guess it's a bug in `make_timeline` too?

Reply:
> I'm not 100% sure off the top of my head. I copy/pasted the timeline from the `make_timeline` tool initially, apparently that doesn't include the reset. If it's still required, I guess it's a bug in `make_timeline` too?

@JLGarber - do you know whether an affirmative Reset line is still required?  As for `make_timeline`, I'm not positive, but I suspect the reason it's not included in the default output is because it wouldn't apply, e.g., to dungeon content or other content that uses seals/unseals instead of ActorControl wipes.  Still, to your point, `make_timeline` should be able to detect if that's the case and add it (if it is, in fact, required).

Reply:
I don't remember a specific change that would have let us remove the actor control line, and I think I'm not in a good spot to review the timeline module to double-check for a few days. Let's leave it in for now maybe and I'll look at the timeline module more closely next week.

Reply:
It looks like most reset syncs were dropped from older content in quisquous/cactbot#5555 by @wexxlee, when those timelines were switched over to starting on `InCombat` lines (ref: quisquous/cactbot#5440); but resets were kept when a similar change was made to Endwalker content in quisquous/cactbot#5783 by @quisquous.

Per https://github.com/quisquous/cactbot/pull/5555#issuecomment-1592031765, it sounds like most reset lines could be dropped, other than content where `resetWhenOutOfCombat: false` is set (Hunts/FATEs, possibly Adventuring Forays).

Reply:
Ah, thanks for tracking that down for us. I think that settles it then, we can leave it out here. I'll look at potentially adding that flag to `make_timeline` next week when things have calmed a little.